script/riiif_cache_clean.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+#
+# riiif_cache_clean.sh
+# Cron script for maintaining the loris cache size.
+#
+# CAUTION - This script deletes files. Be careful where you point it!
+#
+
+LOG="../tmp/riiif_cache_clean.log"
+
+# Check that the cache directories...
+IMG_CACHE_DIR="../tmp/network_files"
+
+# ...is below a certain size...
+REDUCE_TO=0 #0 gb
```

Comment:
@scherztc if REDUCE_TO is set to zero, this script does nothing more than just wipe out everything in the cache directory every time we run it.  So I suggest setting it to 500000 for now so the cache never gets above half a gigabyte.  The most recent cache files up to half a gig will stay in the directory after we run the script.